WindDirectionControl/WindDirectionControl.js

```diff
@@ -0,0 +1,133 @@
+import compose from 'ramda/src/compose';
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Arc from '../Arc';
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+import WindDirectionControlBehaviourDecorator from './WindDirectionControlBehaviourDecorator';
+
+import css from './WindDirectionControl.module.less';
+
+const WIND_DIRECTION = [
+	{option: 'airDown'},
+	{option: 'airRight'},
+	{option: 'airUp'}
+];
+
+/**
+ * A SVG for {@link agate/WindDirectionControl}.
+ *
+ * @class WindDirectionControl
+ * @memberof agate/WindDirectionControl
+ * @ui
+ * @private
+ */
+const WindDirectionControlBase = kind({
+	name: 'WindDirectionControlBase',
+
+	propTypes: /** @lends agate/WindDirectionControl.WindDirectionControlBase.prototype */ {
+		/**
+		 * State of WindDirectionControl.
+		 *
+		 * @type {('airDown'|'airRight'|'airUp')}
+		 * @public
+		 */
+		airDirection: PropTypes.oneOf(['airDown', 'airRight', 'airUp']),
+
+		/**
+		 * Called when the path area is clicked.
+		 *
+		 * @type {Function}
+		 * @param {Object} event
+		 * @public
+		 */
+		onClick: PropTypes.func,
+
+		/**
+		 * Current skinVariant.
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		skinVariants: PropTypes.object
+	},
+
+	styles: {
+		css,
+		className: 'windDirectionControl'
+	},
+
+	computed: {
+		componentIcon: (props) => {
+			const {airDirection} = props;
+
+			switch (airDirection) {
+				case 'airDown':
+					return 'airdown';
+				case 'airRight':
+					return 'airup';
+				case 'airUp':
+					return 'airright';
+			}
+		}
+	},
+
+	render: ({airDirection, componentIcon, onClick, skinVariants, ...rest}) => {
+
+		return (
+			<div className={css.windDirectionControl} {...rest}>
+				{WIND_DIRECTION.map((windDirection, index) => {
```

Comment:
wrap all the <Arc>s into another div, in order to have it centered inside its parent.
Example styling for centering:
	.arcs{
		display: flex;
		align-items: center;
		justify-content: center;
		height: 100%;
		width: 100%;
	}